### PR TITLE
Support for Rebar3 style deps

### PR DESCRIPTION
**Pull request by walrusVision**
Elvis fails on a rebar.config file which uses Rebar3 style deps with the following error.

```
# rebar.config [FAIL]
Error: 'function_clause' while applying rule 'protocol_for_deps_rebar'.
Loading files...
Applying rules...
```

This is the style which it fails on.
```
{deps, [
  {poolboy, {git, "https://github.com/devinus/poolboy.git", {tag, "1.5.1"}}},
  {gproc,   {git, "https://github.com/uwiger/gproc.git", {branch, "master"}}}
]
}.
```

### CODE REVIEW

File: test/project_SUITE.erl [hunk 1]
```diff
@@ -167,16 +167,16 @@ verify_protocol_for_deps_rebar(_Config) ->
     Filename = "rebar.config.fail",
     {ok, File} = elvis_test_utils:find_file(SrcDirs, Filename),
 
-    [_, _] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, #{}),
+    [_, _, _, _] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, #{}),
 
     RuleConfig =  #{ignore => [getopt]},
-    [_] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, RuleConfig),
+    [_, _] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, RuleConfig),
```
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> Line 173 is too long:     [_, _] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, RuleConfig),.

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'getopt' revision is specified 'master', please change this to a tag, branch or specific commit.

File: test/project_SUITE.erl [hunk 2]
```diff
@@ -167,16 +167,16 @@ verify_protocol_for_deps_rebar(_Config) ->
     Filename = "rebar.config.fail",
     {ok, File} = elvis_test_utils:find_file(SrcDirs, Filename),
 
-    [_, _] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, #{}),
+    [_, _, _, _] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, #{}),
```
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> Line 170 is too long:     [_, _, _, _] = elvis_project:protocol_for_deps_rebar(ElvisConfig, File, #{}),.

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'aleppo' revision is specified 'master', please change this to a tag, branch or specific commit.

**Comment by jfacorro** [comment]
@walrusVision Thanks for your PR. Could you check the line length warnings from gadgetci? Just ignore the other ones about a dep being in master since that's just the test file, which is supposed to fail. Thanks!

**Pull request merged**